README.md

```diff
@@ -72,6 +72,10 @@ application, running your extension service has several benefits:
    through more standard and secure channels.
 
 ## Deploying
+Clone this repo to your local machine:
```

Comment:
I don't love this here. A couple suggestions:
* create a before you begin doc with all common instructions
* Create a before you begin section above `[Setting up your Database](https://github.com/GoogleCloudPlatform/database-query-extension/blob/199ec0c657bb02197ee720d6792cd78c4d2ba5b1/README.md#setting-up-your-database)`
* Add this to all docs with the qualifier: "If you haven't already cloned this repo ..."

Reply:
Created a before you being section above [Setting up your database]. Can also move it back to `setting up your database` doc if it makes more sense there.